Source/Core/Core/AchievementManager.h

```diff
@@ -0,0 +1,22 @@
+// Copyright 2023 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#ifdef USE_RETRO_ACHIEVEMENTS
+#include <string>
+
+namespace Achievements
+{
+
+void Init();
+void Login();
+std::string Login(std::string password);
```

Comment:
```suggestion
std::string Login(const std::string& password);
```

seems like this got missed when switching the implementation over

Reply:
I just want to leave a note here that I do have this in a local branch but I'm still working on other larger changes which is why it hasn't been followed up on here yet.